### PR TITLE
added ID

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: EMIP/1-1000/EMIP00130.xml [hunk 1]
```diff
@@ -64,7 +64,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <note>(Budge, IV; Princeton Ethiopic 41, ff. 55v–56r; MYS 1, ff. 7rv).</note>
                     </msItem>
                <msItem xml:id="ms_i2"><locus from="10r" to="24v"/>
-                        <title>Prayer to Mary, “I take refuge.”</title>
+                 <title ref="LIT6161BakwelluH">Prayer to Mary, “I take refuge.”</title>
                         <incipit xml:lang="gez">
                           በኵሉ፡ ሕማሙ፡ ዘዕለተ፡ ዓርብ፡ ለፍቁር፡ ወልድኪ፡ ተማኅፀንኩ፡ ኦ፡ እግዝእትየ፡ ማርያም፡ በእኅዞቱ፡ በመጣብሕ፡...
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00130.xml:69]: You do not need three dots at the end

File: EMIP/1-1000/EMIP00616.xml [hunk 2]
```diff
@@ -94,7 +94,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
                   <textLang mainLang="gez">Gǝ‘ǝz</textLang>
                 </msItem>
-                  <msItem xml:id="ms_i11"><locus from="41r" to="44r" facs="043"/><title type="complete" ref="LIT3017RepCh297"/><incipit xml:lang="gez">በስመ ፡ አብ ፡ ወወልድ ፡ <gap reason="omitted"/> ተማፀንኩ ፡ በኵሉ ፡ ሕማሙ ፡ ዘዕለተ ፡ ዓርብ ፡ ለፍቁር ፡ ወልድኪ። ተማኅፀንኩ ፡ ኦ ፡ እግዝእትየ ፡ ማርያም ፡ በተእኅዞቱ ፡ በመጣብሕ ፡ ወአብር። ተማኅፀንኩ ፡ ኦእግዚእትየ ፡ ማርያም ፡ በበዊኦቱ ፡ ውስተ ፡ ቤተ ፡ ሊቀ ፡ ካህናት ፡ እንዘ ፡ ሕሙይ ፡ ወእሡር ፡ ለፍቀር ፡ ወልድኪ ፡ <gap reason="omitted"/></incipit>
+                 <msItem xml:id="ms_i11"><locus from="41r" to="44r" facs="043"/><title type="complete" ref="LIT6161BakwelluH"/><incipit xml:lang="gez">በስመ ፡ አብ ፡ ወወልድ ፡ <gap reason="omitted"/> ተማፀንኩ ፡ በኵሉ ፡ ሕማሙ ፡ ዘዕለተ ፡ ዓርብ ፡ ለፍቁር ፡ ወልድኪ። ተማኅፀንኩ ፡ ኦ ፡ እግዝእትየ ፡ ማርያም ፡ በተእኅዞቱ ፡ በመጣብሕ ፡ ወአብር። ተማኅፀንኩ ፡ ኦእግዚእትየ ፡ ማርያም ፡ በበዊኦቱ ፡ ውስተ ፡ ቤተ ፡ ሊቀ ፡ ካህናት ፡ እንዘ ፡ ሕሙይ ፡ ወእሡር ፡ ለፍቀር ፡ ወልድኪ ፡ <gap reason="omitted"/></incipit>
                   <textLang mainLang="gez">Gǝ‘ǝz</textLang>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00616.xml:98]: "gez"/>

File: EMIP/1-1000/EMIP00130.xml [hunk 3]
```diff
@@ -64,7 +64,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                         <note>(Budge, IV; Princeton Ethiopic 41, ff. 55v–56r; MYS 1, ff. 7rv).</note>
                     </msItem>
                <msItem xml:id="ms_i2"><locus from="10r" to="24v"/>
-                        <title>Prayer to Mary, “I take refuge.”</title>
+                 <title ref="LIT6161BakwelluH">Prayer to Mary, “I take refuge.”</title>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00130.xml:67]: If the record for work is filled properly, you do not need to include in addition "Prayer to Mary" etc.

File: EMIP/1-1000/EMIP00616.xml [hunk 4]
```diff
@@ -94,7 +94,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
                 </msItem>
-                  <msItem xml:id="ms_i11"><locus from="41r" to="44r" facs="043"/><title type="complete" ref="LIT3017RepCh297"/><incipit xml:lang="gez">በስመ ፡ አብ ፡ ወወልድ ፡ <gap reason="omitted"/> ተማፀንኩ ፡ በኵሉ ፡ ሕማሙ ፡ ዘዕለተ ፡ ዓርብ ፡ ለፍቁር ፡ ወልድኪ። ተማኅፀንኩ ፡ ኦ ፡ እግዝእትየ ፡ ማርያም ፡ በተእኅዞቱ ፡ በመጣብሕ ፡ ወአብር። ተማኅፀንኩ ፡ ኦእግዚእትየ ፡ ማርያም ፡ በበዊኦቱ ፡ ውስተ ፡ ቤተ ፡ ሊቀ ፡ ካህናት ፡ እንዘ ፡ ሕሙይ ፡ ወእሡር ፡ ለፍቀር ፡ ወልድኪ ፡ <gap reason="omitted"/></incipit>
+                 <msItem xml:id="ms_i11"><locus from="41r" to="44r" facs="043"/><title type="complete" ref="LIT6161BakwelluH"/><incipit xml:lang="gez">በስመ ፡ አብ ፡ ወወልድ ፡ <gap reason="omitted"/> ተማፀንኩ ፡ በኵሉ ፡ ሕማሙ ፡ ዘዕለተ ፡ ዓርብ ፡ ለፍቁር ፡ ወልድኪ። ተማኅፀንኩ ፡ ኦ ፡ እግዝእትየ ፡ ማርያም ፡ በተእኅዞቱ ፡ በመጣብሕ ፡ ወአብር። ተማኅፀንኩ ፡ ኦእግዚእትየ ፡ ማርያም ፡ በበዊኦቱ ፡ ውስተ ፡ ቤተ ፡ ሊቀ ፡ ካህናት ፡ እንዘ ፡ ሕሙይ ፡ ወእሡር ፡ ለፍቀር ፡ ወልድኪ ፡ <gap reason="omitted"/></incipit>
```
Comment (DenisNosnitsin1970) [inline EMIP/1-1000/EMIP00616.xml:97]: As far as I understand, at the end of the quotation from the incipit you do not need <gap>. It is used, for instance, if you the text is interrupted with the quotation.

**Comment by thea-m** [comment]
Thank you @DenisNosnitsin1970 , you are of course right, but I just added a newly created ID to some EMIP records, I will not edit them exhaustingly.

File: EMIP/1-1000/EMIP00616.xml (hunk 4 above)
Comment (Ralph-Lee-UK) [reply EMIP/1-1000/EMIP00616.xml:97]: Thanks for clarifying this, I had wondered.

**Pull request merged**